crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -1234,4 +1262,19 @@ mod tests {
 
         assert_eq!(snapshot_actual, snapshot_actual);
     }
+
+    #[tokio::test]
+    async fn resolves_indirect_ref_to_git_https() {
+        let indirect_ref = FlakeRef::from_url("flake:flox", PARSER_UTIL_BIN_PATH).unwrap();
```

Comment:
why not just from_str? I understand that will use the PARSER_UTIL_BIN_PATH available to runix anyway

Reply:
I don't think this matters all that much so I didn't give it much thought one way or the other.